packages/components/src/button/index.js

```diff
@@ -30,6 +35,7 @@ export function Button( props, ref ) {
 		isSecondary,
 		isLink,
 		isDestructive,
+		isFocusable,
```

Comment:
Should this default to true instead? I feel this was needed in every disabled button we have in the UI. I'm even wondering if we need a prop or just do this automatically.

Reply:
Now, it makes me wonder how often we want to have a button which is both disabled and not focusable at the same time. It feels like for accessibility reasons it should always be focusable.

https://ux.stackexchange.com/questions/103239/should-disabled-elements-be-focusable-for-accessibility-purposes - it only confirms this reasoning.

It might be a good idea to set isFocusable to true by default as suggested.

We might be some disabled and not focusable buttons but it might be simply wrong. We should found those cases and take further steps based on that.

Reply:
Made `isFocusable` true by default